internal/cli/organizations/create.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const createTemplate = `Organization {{.ID}} created.`
```

Comment:
you either do
```suggestion
const createTemplate = "Organization {{.ID}} created.\n"
```
Or
```go
const createTemplate = `Organization {{.ID}} created.
`
```
(text blocks you need an actual new line